Contributing.md

```diff
@@ -0,0 +1,70 @@
+# Introduction:
+First off, thank you for considering contributing to SQL Server Time Zone Support.  If you're passionate about Time zones, passionate about SQL Server, or you just want to make the SQL world a little better, you're in the right place.  
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, the maintainers will reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+We're looking for and eagerly accepting all contributions.  Whether you want to improve performance or fix bugs, or improve documentation,  triage bugs, or write  tutorials, any contribution is welcome.  
+
+
+## Ground Rules
+> Content to be added
+
+## Your First Contribution
+
+Unsure where to begin contributing? You can start by looking through these beginner and help-wanted issues: 
```

Comment:
I would simplify this to "Unsure where to begin contribution? Look for issues with an up-for-grabs or help-wanted tag"